### DISCUSSION TITLE
Export pipeline into a python file

### DISCUSSION THREAD

**Discussion by arita37** [skrub-data/skrub#732] [Ideas]
Exporting the tableVectorizer pipeline into a static file would be very useful.

Tablevectorizer --> easy experimenet
python file of pipeline --> stable runs.

At the end, we need to review "manually" the pipeline to see any issues.

2) tpot has good exporter of pipelines
https://github.com/EpistasisLab/tpot/blob/master/tpot/export_utils.py

**Comment by GaelVaroquaux** [comment]
This is not in the scope of skrub. It is more related to MLops than to data prep.

It could however be in the scope of skops, however: https://github.com/skops-dev/skops
